ios/cpp/PytorchExp.podspec

```diff
@@ -0,0 +1,38 @@
+Pod::Spec.new do |s|
+    s.name             = 'Pytorch'
+    s.version          = '0.0.1'
+    s.authors          = 'Facebook'
```

Comment:
Can we change this to 'PyTorch Developers'  cc @soumith

Reply:
yea, `PyTorch Team` sounds good